resources/cluster-essentials/templates/priorityclass.yaml

```diff
@@ -0,0 +1,9 @@
+{{- if .Values.global.priorityClassName }}
+apiVersion: scheduling.k8s.io/v1
+kind: PriorityClass
+metadata:
+  name: {{ .Values.global.priorityClassName }}
+value: {{ .Values.global.priorityClassValue }}
+globalDefault: false
+description: "Default scheduling priority of Kyma installer. Must not be blocked by unschedulable user or Kyma workloads."
```

Comment:
The description is wrong. It's default PriorityClass for all Kyma components but the Installer. Installer has it's own class named `kyma-installer`

Reply:
Copy-paste power! :)